omod-common/src/main/java/org/openmrs/module/webservices/rest/web/ConversionUtil.java

```diff
@@ -504,5 +505,70 @@ public static SimpleObject getAuditInfo(Object delegate) {
 		
 		return ret;
 	}
-	
```

Comment:
This is moved here to a public static method in ConversionUtil without any changes, previously was a private instance method on BaseDelegatingResource